src/config/debugbar.php

```diff
@@ -26,28 +149,29 @@
      */
 
     'collectors' => [
-        'phpinfo' => true,  // Php version
-        'memory' => true,  // Memory usage
-        'exceptions' => true,  // Exception displayer
-        'db' => true,  // Show database (PDO) queries and bindings
-        'laravel' => true, // Laravel version and environment
-
-        'events' => false, // All events fired
-        'log' => false,  // Logs from Monolog (merged in messages if enabled)
-        'messages' => false,  // Messages
-        'time' => false,  // Time Datalogger
-        'views' => false,  // Views with their data
-        'route' => false,  // Current route information
+        'phpinfo' => true, // Php version
+        'messages' => false, // Messages
+        'time' => true, // Time Datalogger
+        'memory' => true, // Memory usage
+        'exceptions' => true, // Exception displayer
+        'log' => false, // Logs from Monolog (merged in messages if enabled)
+        'db' => true, // Show database (PDO) queries and bindings
+        'views' => false, // Views with their data
+        'route' => false, // Current route information
         'auth' => false, // Display Laravel authentication status
         'gate' => false, // Display Laravel Gate checks
-        'session' => false,  // Display session data
-        'symfony_request' => false,  // Only one can be enabled..
-        'mail' => false,  // Catch mail messages
+        'session' => false, // Display session data
+        'symfony_request' => false, // Only one can be enabled..
+        'mail' => false, // Catch mail messages
+        'laravel' => true, // Laravel version and environment
+        'events' => false, // All events fired
         'default_request' => false, // Regular or special Symfony request logger
         'logs' => false, // Add the latest log messages
         'files' => false, // Show the included files
         'config' => false, // Display config settings
         'cache' => false, // Display cache events
+        'models' => false, // Display models
```

Comment:
Can we enable this by default? I think it has been included before but might have been split out from `db` in newer versions.

It shows how many instances of each model you have loaded during the request and has been very useful.